src/Ilios/WebBundle/Controller/IndexController.php

```diff
@@ -38,19 +38,23 @@ public function __construct(Filesystem $fs, EngineInterface $templatingEngine)
     /**
      * Respond to GET requests
      *
-     * @return Response
+     * @param Request $request
+     * @param $fileName
+     * @param $versionedStaticFile
+     *
+     * @return BinaryFileResponse|Response
      */
-    public function getAction(Request $request, $fileName)
+    public function getAction(Request $request, $fileName, $versionedStaticFile)
```

Comment:
suggesting to default `$versionedStaticFile` to `FALSE`, then omit setting explicit values for this flag in `routing.yml`.